run

```diff
@@ -43,34 +61,34 @@ print_help() {
 setup_dir() {
     for target in "${postgres_targets[@]}"; do
         mkdir -p "${sysroot}"/lib/rustlib/"${target}"/lib
-        cp --verbose ./"${target}.json" "${sysroot}"/lib/rustlib/"${target}"/target.json
+        cp -f -v ./"${target}.json" "${sysroot}"/lib/rustlib/"${target}"/target.json
     done
 }
 
 copy_json() {
     for target in "${postgres_targets[@]}"; do
-        cp --verbose ./"${target}.json" "${sysroot}"/lib/rustlib/"${target}"/target.json
+        cp -f -v ./"${target}.json" "${sysroot}"/lib/rustlib/"${target}"/target.json
     done
 }
 
 remove_rlibs() {
     for target in "${postgres_targets[@]}"; do
-        rm --force --verbose "${sysroot}"/lib/rustlib/"${target}"/lib/*.rlib
+        rm -f -v "${sysroot}"/lib/rustlib/"${target}"/lib/*.rlib
     done
 }
 
 copy_rlibs() {
     for target in "${postgres_targets[@]}"; do
-        cp --force --verbose target/"${target}"/release/deps/*.rlib "${sysroot}/lib/rustlib/${target}/lib"
+        cp -f -v target/"${target}"/release/deps/*.rlib "${sysroot}/lib/rustlib/${target}/lib"
```

Comment:
...are these longargs not supported by the macOS `cp`?
What bottom of which barrel did they find it?

Reply:
Indeed. Some old BSD.